src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int numberOfBall = 3;
+    private static final Lottery lottery = new Lottery();
```

Comment:
services and objects are usually not constants but class `private final` fields
```suggestion
    private final Lottery lottery = new Lottery();
```

Reply:
not fixed

Reply:
not fixed : make lottery the main-method field.